locales/en/allocation.json

```diff
@@ -14,6 +14,11 @@
     "content": "<p>As this person needs a special vehicle they cannot be part of this allocation.</p>\n<p>If you would still like to move this person you can create a single request for them to move to another prison.",
     "return_link_text": "Return to allocation"
   },
+  "assign": {
+    "sidebar": {
+      "heading": "Prisoner to be allocated"
```

Comment:
There is [already a key](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/locales/en/default.json#L20) for this value that we should use to be consistent.

I think we may tweak that content in a future PR to use `prisoner` instead of `person` but for this  change we should use the same key so we avoid having to maintain that copy in two places.